config/canvas.php

```diff
@@ -1,6 +1,16 @@
 <?php
 
 return [
+    /*
+    |--------------------------------------------------------------------------
+    | Base Domain
+    |--------------------------------------------------------------------------
+    |
+    | This is the domain where Canvas will be accessible from. You are free to
+    | to change this domain to a sub-domain you like.
+    |
+    */
+    'domain' => env('CANVAS_PATH_DOMAIN', env('APP_URL')),
```

Comment:
Let's change the environment variable to `CANVAS_ROUTE_DOMAIN`

Reply:
Okay